### PR TITLE
Kostas/ops 8836 Create 'You cannot set up a Direct Debit online' page

### PR DESCRIPTION
**Pull request by Kntanos**
Page is now linked to the previous one.

TODO href for "Back to tax account" button. I don't know what the solution for this will be. I suggest a new ticket is created.

### CODE REVIEW

File: app/assets/stylesheets/self-service-time-to-pay.css [hunk 1]
```diff
@@ -376,4 +376,8 @@ details summary:focus,
 
 .details-margin {
     margin-bottom:20px;
+}
+
+span.keeptogether {
```
Comment (darrell-plant) [inline app/assets/stylesheets/self-service-time-to-pay.css]: you could call it ".nowrap" or ".no-wrap" maybe?

File: app/views/core/not_sole_signatory.scala.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+@(
+    isWelsh: Boolean,
+    loggedIn: Boolean = false)(implicit
+    messages: play.api.i18n.Messages,
+    request: Request[_],
+    appConfig: config.AppConfig,
+)
+
+@main(
+    title = Messages("ssttp.not-sole-signatory.title"),
```
Comment (darrell-plant) [inline app/views/core/not_sole_signatory.scala.html]: can you change `Messages`  to `messages` in this file pls

File: app/assets/stylesheets/self-service-time-to-pay.css (hunk 1 above)
Comment (Kntanos) [reply app/assets/stylesheets/self-service-time-to-pay.css]: done

File: app/views/core/not_sole_signatory.scala.html (hunk 2 above)
Comment (Kntanos) [reply app/views/core/not_sole_signatory.scala.html]: done

**Review by darrell-plant — APPROVED** [review]

**Pull request merged**